src/Evolution/Systems/Ccz4/Ricci.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
```

Comment:
this shouldn't be in the GR group. It's CCZ4-specific while the GR group is generic. This code should be findable by going `Code Reference->Topics->Evolution Systems->Ccz4`. I think that namespace would normally be documented in `Evolution/Systems/Ccz4/System.hpp`. If you want, you can add that file with just documenting the namespace. This would also mean we wouldn't need to use the search functionality to find the documentation 🙃